syntax = "proto3";
// 解析数据的数据格式
// 消息通知
message SpiderMessage {
  enum SpiderStatus {
    // 刚创建还未解析的时候
    NONE = 0;
    // 需要重试的
    RETRY = 1;
    // 已经解析
    PARSED = 2;
  }
  // 唯一id
  string id = 1;
  // 创建时间戳
  string create_time = 2;
  SpiderStatus status = 3;
  string request_url = 4;
  // 返回的消息
  string response = 5;
  // 重试时间
  string retry_time = 6;
  // 解析时间
  string parse_time = 7;
}
